app/models/follow_request.rb

```diff
@@ -27,7 +27,5 @@ def authorize!
     destroy!
   end
 
-  def reject!
-    destroy!
-  end
+  alias reject! destroy!
```

Comment:
`alias :reject!, :destory!` ?

Reply:
> [When aliasing a method, you can use either its name or a symbol:](https://docs.ruby-lang.org/en/2.4.0/syntax/miscellaneous_rdoc.html#label-alias)

I actually don't care either to use bareword(reject!) or symbol(:reject), but RuboCop asked me to use bareword.

Do you want me to use symbol?